src/main/js/page_objects/blueocean/bluePipelineRunDetail.js

```diff
@@ -95,14 +117,55 @@ module.exports.commands = [{
         this.expect.element('@code').to.not.be.present.before(1000);
 
     },
+    validateGraph: function () {
+        var self = this;
+        self.waitForElementVisible('@highlightedGraph');
+        return self;
+    },
+    validateSteps: function (browser) {
+        var self = this;
+        self.waitForElementVisible('@logConsole');
+        browser.elements('css selector', '.logConsole', function (codeCollection) {
```

Comment:
The problem is some methods are not exposed in ```self``` browser brings:

- url
- elemets
- ....

I only pass browser when I need those methods, there maybe another way which fixes as well the problem with the selectors not being able to be used in those ```browser methods```